semantic-java/src/Language/Java/Tags.hs

```diff
@@ -0,0 +1,103 @@
+{-# LANGUAGE AllowAmbiguousTypes, DataKinds, DisambiguateRecordFields, FlexibleContexts, FlexibleInstances, MultiParamTypeClasses, NamedFieldPuns, ScopedTypeVariables, TypeApplications, TypeFamilies, TypeOperators, UndecidableInstances #-}
+module Language.Java.Tags
```

Comment:
Note that this is incomplete in the sense that since precise ASTs can’t currently represent comments, we can’t collect doc comments.